app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailFragment.kt

```diff
@@ -77,13 +78,29 @@ class SessionDetailFragment : Fragment(), Injectable {
         binding.toolbar.setNavigationOnClickListener { activity?.finish() }
     }
 
-    private fun bindSession(session: Session.SpeechSession) {
+    private fun bindSession(session: Session.SpeechSession, sessionId: String) {
         binding.session = session
         binding.fab.setOnClickListener {
             updateDrawable()
             sessionDetailViewModel.onFavoriteClick(session)
             sessionAlarm.toggleRegister(session)
         }
+        binding.fabShare.setOnClickListener {
+            val shareIntent = Intent(Intent.ACTION_SEND).apply {
+                type = "text/plain"
+                var speakerList = listOf<String>()
+                session.speakers.forEach { speakerList += it.name }
+                val speakers = speakerList.joinToString(", ")
+                val title = resources.getString(R.string.session_share_title,
+                        session.title,
+                        session.room.name,
+                        speakers)
+                val url = resources.getString(R.string.session_share_url, sessionId)
+                putExtra(Intent.EXTRA_TEXT, "$title\n$url")
```

Comment:
example
> ConstraintLayout, now and future at Room 6 by Nicolas Roard, John Hoford, thagikura
https://droidkaigi.jp/2018/timetable?session=16988

Reply:
[Q] Should I add a hashtag?

Reply:
Please wait. I will talk with staff🙏

Reply:
The staff has decided to `#droidkaigi_room1`...`#droidkaigi_room7` `#droidkaigi_hall`.

Reply:
:pray:

Reply:
I added

e.g.
> ConstraintLayout, now and future (Nicolas Roard, John Hoford, thagikura) #droidkaigi_room6
https://droidkaigi.jp/2018/timetable/?session=16988